packages/react-component-library/src/components/Timeline/Event.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import classNames from 'classnames'
+import { format } from 'date-fns'
+
+import { DAY_WIDTH } from './constants'
+import { useTimelinePosition } from './hooks/useTimelinePosition'
+
+export interface EventProps extends ComponentWithClass {
+  startDate: Date
+  endDate: Date
+  status?: string
+  title?: string
+}
+
+export const Event: React.FC<EventProps> = ({
+  startDate,
+  endDate,
+  status,
+  title,
+}) => {
+  const { offset, width, isBeforeStart, isAfterEnd } = useTimelinePosition(
+    startDate,
+    endDate
+  )
+
+  if (isBeforeStart || isAfterEnd) return null
+
+  const classes = classNames('timeline__event', {
+    [`timeline__event--${status ? status.toLowerCase() : ''}`]: status,
```

Comment:
It might make this easier to read if `status` is defaulted to empty string in the function arguments.

Reply:
To be honest I think the ternary statement is not required as it will have the same affect if `status` is not set.